src/java.base/share/classes/javax/crypto/Cipher.java

```diff
@@ -601,13 +601,15 @@ public static final Cipher getInstance(String transformation)
      * @throws NoSuchAlgorithmException if {@code transformation}
      *         is {@code null}, empty, in an invalid format,
      *         or if a {@code CipherSpi} implementation for the
-     *         specified algorithm is not available from the specified
-     *         provider
+     *         specified {@code transformation} is not available from
+     *         the {@code provider} due to causes not covered by the
+     *         {@code NoSuchPaddingException}
      *
-     * @throws NoSuchPaddingException if {@code transformation}
-     *         contains a padding scheme that is not available
+     * @throws NoSuchPaddingException if a {@code CipherSpi} object
+     *         from the {@code provider} is found using the algorithm
```

Comment:
Suggest to s/from the/from the specified, and to make the same changes in this method at lines #567 and #568, s/the specified provider/the specified {@code provider}.

Reply:
Sure, will change~